airbyte-cdk/python/airbyte_cdk/test/catalog_builder.py

```diff
@@ -45,10 +49,12 @@ def with_stream(self, name: ConfiguredAirbyteStreamBuilder) -> "CatalogBuilder":
         ...
 
     @overload
-    def with_stream(self, name: str, sync_mode: SyncMode) -> "CatalogBuilder":
+    def with_stream(self, name: str, sync_mode: SyncMode, json_schema: Dict[str, Any]) -> "CatalogBuilder":
         ...
 
-    def with_stream(self, name: Union[str, ConfiguredAirbyteStreamBuilder], sync_mode: Union[SyncMode, None] = None) -> "CatalogBuilder":
+    def with_stream(
```

Comment:
The comments below this method mentions that we are trying to deprecate this method. Should we therefore not add functionality to it?

Reply:
Sure, I will rely now directly on `ConfiguredAirbyteStreamBuilder`, I already removed the new functionality.